proto/osmosis/poolmanager/v1beta1/query.proto

```diff
@@ -27,13 +27,27 @@ service Query {
         "/osmosis/gamm/v1beta1/{pool_id}/estimate/swap_exact_amount_in";
   }
 
+  rpc EstimateSinglePoolSwapExactAmountIn(
+      EstimateSinglePoolSwapExactAmountInRequest)
+      returns (EstimateSwapExactAmountInResponse) {
+    option (google.api.http).get = "/osmosis/gamm/v1beta1/{pool_id}/estimate/"
```

Comment:
```suggestion
    option (google.api.http).get = "/osmosis/poolmanager/v1beta1/{pool_id}/estimate/"
```

Reply:
ohh good catch on all of these! committing them